### PR TITLE
feat: add keyboard handling docs

### PR DESCRIPTION
**Pull request by dobrinyonkov**
Fixes: #3089

### CODE REVIEW

File: packages/fiori/src/FlexibleColumnLayout.js [hunk 1]
```diff
@@ -231,6 +231,14 @@ const metadata = {
  * The component would display 1 column for window size smaller than 599px, up to two columns between 599px and 1023px,
  * and 3 columns for sizes bigger than 1023px.
  *
+ * <br><br>
+ * <h3>Keyboard Handling</h3>
+ *
+ * <ul>
+ * <li>[SPACE, ENTER] - If focus is on the layout toggle button (arrow button) trigger the associated action (such as expand/collapse the column).</li>
```
Comment (fifoosid) [inline packages/fiori/src/FlexibleColumnLayout.js:238]: If focus is on the layout toggle button (arrow button), once activated, it triggers the associated action (such as expand/collapse the column).

File: packages/main/src/Avatar.js [hunk 2]
```diff
@@ -226,6 +226,15 @@ const metadata = {
  * The shape can be circular or square. There are several predefined sizes, as well as an option to
  * set a custom size.
  *
+ * <br><br>
+ * <h3>Keyboard Handling</h3>
+ *
+ * <ul>
+ * <li>[SPACE, ENTER] - Fires the <code>ui5-click</code> event if the <code>interactive</code> property is set to true.</li>
```
Comment (fifoosid) [inline packages/main/src/Avatar.js:233]: [SPACE, ENTER, RETURN]

File: packages/main/src/Avatar.js [hunk 3]
```diff
@@ -226,6 +226,15 @@ const metadata = {
...
+ * <li>[SHIFT] - If [SPACE] or [ENTER],[RETURN] is pressed, releases the <code>ui5-avatar</code> without triggering the click event.</li>
```
Comment (fifoosid) [inline packages/main/src/Avatar.js:234]: If [SPACE] or [ENTER],[RETURN] is pressed, pressing [SHIFT] releases the <code>ui5-avatar</code> without triggering the click event.

File: packages/main/src/Icon.js [hunk 4]
```diff
@@ -155,6 +155,15 @@ const metadata = {
  * <br>
  * <code>import "@ui5/webcomponents-icons-tnt/dist/antenna.js";</code>
  *
+ * <br><br>
+ * <h3>Keyboard Handling</h3>
+ *
+ * <ul>
+ * <li>[SPACE, ENTER] - Fires the <code>ui5-click</code> event if the <code>interactive</code> property is set to true.</li>
+ * <li>[SHIFT] - If [SPACE] or [ENTER],[RETURN] is pressed, releases the <code>ui5-icon</code> without triggering the click event.</li>
```
Comment (fifoosid) [inline packages/main/src/Icon.js:163]: If [SPACE] or [ENTER],[RETURN] is pressed, pressing [SHIFT] releases the <code>ui5-icon</code> without triggering the click event.

**Review by fifoosid — CHANGES_REQUESTED** [review]

**Review by ilhan007 — CHANGES_REQUESTED** [review]

**Review by ilhan007 — APPROVED** [review]

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**